MantidQt/CustomInterfaces/src/EnggDiffraction/EnggDiffractionPresenter.cpp

```diff
@@ -277,6 +345,13 @@ void EnggDiffractionPresenter::processFocusTexture() {
 
   } else if (focusMode == 1) {
     g_log.debug() << " focus mode selected Focus Sum Of Files " << std::endl;
```

Comment:
debug might be a bit too low level for this (you get an extremely verbose log with debug). I'd go for information or even notice, but that's just personal taste.